Shop_Unity_base/Assets/Scripts/Shop/Model/ShopModel.cs

```
@@ -2,166 +2,182 @@
 
-        //------------------------------------------------------------------------------------------------------------------------
-        //                                                  GetSelectedItem()
-        //------------------------------------------------------------------------------------------------------------------------        
-        //returns the selected item
+        // Returns the selected item
         public Item GetSelectedItem()
         {
             if (selectedItemIndex >= 0 && selectedItemIndex < itemList.Count)
-            {
                 return itemList[selectedItemIndex];
-            }
             else
-            {
                 return null;
-            }
         }
 
-        //------------------------------------------------------------------------------------------------------------------------
-        //                                                  SelectItem()
-        //------------------------------------------------------------------------------------------------------------------------
-        //attempts to select the given item, fails silently
+        // Attempts to select the given item, fails silently
         public void SelectItem(Item item)
         {
             if (item != null)
             {
                 int index = itemList.IndexOf(item);
                 if (index >= 0)
-                {
                     selectedItemIndex = index;
-                }
             }
         }
 
-        //------------------------------------------------------------------------------------------------------------------------
-        //                                                  SelectItemByIndex()
-        //------------------------------------------------------------------------------------------------------------------------        
-        //attempts to select the item, specified by 'index', fails silently
+        // Attempts to select the item, specified by 'index', fails silently
```

Same as the previous comment on "failing silently".